webpack/webpack.bzl

```diff
@@ -0,0 +1,142 @@
+"""Webpack bundle producing rule defintion."""
+
+load("@build_bazel_rules_nodejs//:providers.bzl", "DeclarationInfo", "ExternalNpmPackageInfo", "JSModuleInfo", "node_modules_aspect", "run_node")
+load("@build_bazel_rules_nodejs//internal/common:expand_variables.bzl", "expand_variables")
+load("@build_bazel_rules_nodejs//internal/linker:link_node_modules.bzl", "module_mappings_aspect")
+
+
+def get_worker_bin():
+    # BEGIN-INTERNAL
+    return Label("//webpack/bin:webpack-worker")
+    # END-INTERNAL
+    return Label("//@bazel/webpack/bin:webpack-worker")
```

Comment:
we should just add a `@bazel` directory path segment so that no replacements are needed, the path in our repo is the same as the path in a users `npm` external repo

Reply:
So you are suggesting that we create a directory called `@bazel/worker/bin` and put the BUILD file into that directory?  That makes sense.  

One thing that is unclear to be; if the users are not going to install the rule via npm,  do we even need to have different binaries?